pkg/util/inplaceupdate/inplace_utils.go

```diff
@@ -327,6 +332,13 @@ func calculateInPlaceUpdateSpec(oldRevision, newRevision *apps.ControllerRevisio
 		Revision:        newRevision.Name,
 		ContainerImages: make(map[string]string),
 	}
+
+	if opts != nil && opts.CustomizeUpdateFunc != nil {
```

Comment:
Maybe we should leave a comment here. The behavior is not consistent with the calculateInPlaceUpdateSpec method comment above.